CMakeLists.txt

```diff
@@ -289,6 +313,9 @@ elseif(UNIX)
   endif()
 endif()
 
+# Add public headers to sources of PortAudio (used by some IDEs to list them in project tree)
+target_sources(PortAudio PRIVATE "${PORTAUDIO_PUBLIC_HEADERS}")
```

Comment:
How about to try to additionally group public headers into `Public Header Files` group in IDE, will CMake create Public Header Files in your case?:

```cmake
source_group("Public Header Files" FILES ${PORTAUDIO_PUBLIC_HEADERS})
```

Reply:
Yes, it is shown as `Public Header Files` in IDE, I've added that to the PR.
![image](https://user-images.githubusercontent.com/5435069/130367428-e268ff52-910c-4bf7-bb07-2347776e25f1.png)